.gitignore

```diff
@@ -576,3 +576,5 @@ healthchecksdb
 
 # dist for webpack output
 dist
+web
```

Comment:
What is the web folder?

Reply:
`dist` gets renamed to `web` as part of the build process; this just prevents it showing up if the build fails mid-way through.